apispec/ext/marshmallow/swagger.py

```diff
@@ -41,6 +41,32 @@
     marshmallow.fields.List: ('array', None),
 }
 
+CUSTOM_FIELD_MAPPING_ATTR = '__swagger_field_mapping'
+
+
+def map_to_swagger_field(*args):
```

Comment:
I chose "field" b/c the swagger obj it's mapped to has 2 keys, type & format. Still, I don't suppose it matters, so I'll change it rather than prolonging the discussion.